ts/Studio.ts

```diff
@@ -1305,6 +1319,163 @@ export class Studio {
         }
     }
 
+    private bulkFrames(table: HTMLTableElement, framesButton: HTMLButtonElement) {
+        this.setTitle('Bulk Frames');
+
+        let thead = document.createElement('thead');
+        table.appendChild(thead);
+        let tr = document.createElement('tr');
+        thead.appendChild(tr);
+        for (const columnHeader of ['Action', 'Input']) {
+            const th = document.createElement('th');
+            th.textContent = columnHeader;
+            tr.appendChild(th);
+        }
+        const tbody = document.createElement('tbody');
+        table.appendChild(tbody);
+
+        // Move Frame
+        tr = document.createElement('tr');
+        tbody.appendChild(tr);
+        let td = document.createElement('td');
+        td.textContent = 'Move Frame';
+        tr.appendChild(td);
+        td = document.createElement('td');
+        let table2 = document.createElement('table');
+        table2.classList.add('table', 'table-borderless', 'mb-0');
+        td.appendChild(table2);
+        tr.appendChild(td);
+
+        // Move Frame Actions
+        const liftFrames = (value: number) => {
+            this.railroad.frames.forEach((frame: Frame) => {
+                frame.location.z += value;
+            });
+            framesButton.dispatchEvent(new Event('click'));
```

Comment:
Is this intended? This causes the tab to change when clicking the button, which feels pretty weird because it isn't something we do anywhere else in RS.

Reply:
It is intended. I thought that it would make sense here, as its very uncommon that you would need to do multiple actions on that page. And that way you go straight back to frames to see the result. If you feel differently i can remove it.